src/end-chat/end-chat-styles.tsx

```diff
@@ -18,56 +18,52 @@ import * as FlexWebChat from '@twilio/flex-webchat-ui';
 
 const { styled } = FlexWebChat;
 
-export const EndChatWrapper = styled('span')`
+export const ButtonsWrapper = styled('div')`
+  display: flex;
+  text-align: center;
+  margin: -20px auto auto auto;
+  padding-top: 25px;
   width: 100%;
 `;
 
-export const IconWrapper = styled('div')`
-  position: relative;
-  left: 5%;
-`;
-
 export const StyledEndButton = styled('button')`
-  width: 90%;
-  display: flex;
+  align-items: center;
   background-color: #fff;
-  color: #d22f2f;
-  height: 29px;
-  font-weight: bold;
   border: 1.5px solid #d22f2f;
   border-radius: 4px;
+  color: #d22f2f;
   cursor: pointer;
+  display: flex;
   font-family: Open Sans;
   font-size: 12px;
-  margin-left: 5px;
+  font-weight: bold;
   height: 29px;
-  line-height: 24px;
+  margin-left: 5px;
+  width: 90%;
 `;
 
-export const ButtonWrapper = styled('div')`
-  display: flex;
+export const EndChatText = styled('span')`
+  flex-grow: 1;
   text-align: center;
-  margin: -20px auto auto auto;
-  padding-top: 25px;
+`;
+
+export const EndChatIcon = styled('span')`
+  position: relative;
+  left: 7%;
+`;
+
+export const ExitWrapper = styled('div')`
   width: 100%;
 `;
 
-export const ExitDescText = styled('p')`
-  color: #d22f2f;
-  font-family: Open Sans;
-  font-weight: bold;
-  margin: 2px 0 0 1px;
-  font-size: 11px;
-  &:after {
-    content: '';
-    display: inline-block;
-    width: 100%;
-  }
+export const QuickExitText = styled('span')`
+  position: relative;
+  top: -20%;
+  left: -7%;
```

Comment:
I would do it very differently here: instead of positioning using `top` and `left` percentages, I would change the wrapper button to be:
```
display: flex;
align-items: center;
justify-content: center;
```
And maybe `QuickExitText` just needed a `margin-right`.

But I know this code is not originally from your PR, so you don't need to change it now. Just making a note for our future selves.